src/components/Loader.js

```diff
@@ -0,0 +1,20 @@
+import Loader from "react-loader-spinner";
+import React from "react";
+const Loading = () => {
+  return (
+    <Loader
+      style={style.LoadBlock}
+      type="Puff"
+      color="#00BFFF"
+      height={100}
+      width={100}
+    />
+  );
+};
+const style = {
```

Comment:
I would use CSS file here because App already used CSS file so it's better to use only 1 approach